src/pyne.cc

```diff
@@ -10339,7 +10339,7 @@ const double pyne::barns_per_cm2 = 1e24;
 const double pyne::cm2_per_barn = 1e-24;
 const double pyne::sec_per_day = 24.0 * 3600.0;
 const double pyne::MeV_per_K = 8.617343e-11;
-const double pyne::MeV_per_MJ = 1.60217657e-19;
+const double pyne::MeV_per_MJ = 6.2415096471204E+18;
```

Comment:
I don't want to be picky but on translatorscafe.com I got: 6.241506363094e+18.

it is probably better to be consistant with PyNE...